src/output/plugins/PipeWireOutputPlugin.cxx

```diff
@@ -311,6 +311,9 @@ PipeWireOutput::PipeWireOutput(const ConfigBlock &block)
 void
 PipeWireOutput::SetVolume(float _volume)
 {
+	if (thread_loop == nullptr)
```

Comment:
Did you ever observe that crash, or is this just a theory?

Reply:
I have. Setting the volume before playing anything can reproduce it consistently.

```
[i] ~$ mpc clear
volume: 81%   repeat: off   random: off   single: off   consume: off
(reverse-i-search)`': ^C
[i] ~ 130 $ systemctl --user restart mpd.service
[i] ~$ mpc volume 100
MPD error: Connection closed by the server
[i] ~ 1 $
```

Reply:
`output_mixer_set_volume()` checks whether the output is enabled. This must be a bug elsewhere, and your change is just a bad workaround.